### PR TITLE
 tests: add a script to run autopkgtests in the ppa 

### PR DESCRIPTION
**Pull request by elopio**
- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] If this is a bugfix. Have you checked that there is a bug report open for the issue you are trying to fix on [bug reports](https://bugs.launchpad.net/snapcraft)?
- [x] If this is a new feature. Have you discussed the design on the [forum](https://forum.snapcraft.io)?
- [x] Have you successfully run `./runtests.sh static`?
- [x] Have you successfully run `./runtests.sh unit`?

-----
Requires #1643

### CODE REVIEW

File: tools/travis/run_autopkgtests.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+lxc="/snap/bin/lxc"
+
+script_path="$(dirname "$0")"
+project_path="$(readlink -f "$script_path/../..")"
+
+"$script_path/setup_lxd.sh"
+"$script_path/run_lxd_container.sh" test-runner
+$lxc file push --recursive $project_path test-runner/root/
+# Workaround for
+# - Setup snap "core" (2462) security profiles (cannot reload udev rules: exit status 2
```
Comment (sergiusens) [inline tools/travis/run_autopkgtests.sh]: this comment is out of context, should explain its use twice below

Comment (elopio) [reply tools/travis/run_autopkgtests.sh]: On #1643 I've duplicated the comment and moved it down. Is that what you meant?

Comment (sergiusens) [reply tools/travis/run_autopkgtests.sh]: yes

**Comment by elopio** [comment]
Thanks!

**Pull request merged**